feature_layers/create-mobile-geodatabase/src/main/java/com/esri/samples/create_mobile_geodatabase/CreateMobileGeodatabaseController.java

```diff
@@ -0,0 +1,336 @@
+        geodatabaseFeatureTableFuture.addDoneListener(() -> {
+          try {
+            // get the result of the loaded "LocationHistory" table
+            geodatabaseFeatureTable = geodatabaseFeatureTableFuture.get();
+            // create a feature layer for the map using the GeodatabaseFeatureTable
+            var featureLayer = new FeatureLayer(geodatabaseFeatureTable);
+            map.getOperationalLayers().add(featureLayer);
+            createGeodatabaseButton.setDisable(true);
+            closeGeodatabaseButton.setDisable(false);
+            viewTableButton.setDisable(false);
+            mapView.setDisable(false);
+            label.setText("Click map to add features");
+
+          } catch (Exception exception) {
+            new Alert(Alert.AlertType.ERROR, "Failed to get feature table result").show();
+            exception.printStackTrace();
+          }
+        });
+
+      } catch (Exception ex) {
+        new Alert(Alert.AlertType.ERROR, "Failed to get geodatabase result").show();
+        ex.printStackTrace();          }
+    });
+  }
+
+  /**
+   * Closes the geodatabase, displays its directory, and updates UI components.
+   */
+  @FXML
+  private void handleCloseGeodatabase() {
+
+    // close geodatabase, table, and clear input list
+    geodatabase.close();
+    inputs.clear();
+    tableStage.close();
+
+    // display geodatabase file location
+    Alert dialog = new Alert(Alert.AlertType.INFORMATION,
+      "Mobile geodatabase has been closed and saved in the following directory: " + geodatabasePath.toString());
+    dialog.initOwner(mapView.getScene().getWindow());
+    dialog.setHeaderText(null);
+    dialog.setTitle(("Information"));
+    dialog.showAndWait();
+
+    // handle UI
+    viewTableButton.setDisable(true);
+    closeGeodatabaseButton.setDisable(true);
+    createGeodatabaseButton.setDisable(false);
+    label.setText("");
+    graphicsOverlay.getGraphics().clear();
+    map.getOperationalLayers().clear();
+  }
+
+  /**
+   * Displays a new window with the table of features stored in the geodatabase feature table.
+   */
+  @FXML
+  private void handleDisplayTable() {
+
+    // close previous table
+    handleTableWindowVisibility();
+
+    // create observable list of type GeoFeature to store the geodatabase features
+    final ObservableList<FeatureAttributeField> fieldData = FXCollections.observableArrayList();
+
+    // query all the features loaded to the table
+    var queryResultFuture = geodatabaseFeatureTable.queryFeaturesAsync(new QueryParameters());
+    queryResultFuture.addDoneListener(() -> {
+      try {
+        var queryResults = queryResultFuture.get();
+        queryResults.forEach(feature ->
+          // add features to the observable list
+          fieldData.add(new FeatureAttributeField(feature.getAttributes().get("oid").toString(), feature.getAttributes().get("collection_timestamp").toString())));
```

Comment:
```suggestion
          fieldData.add(
          new FeatureAttributeField(feature.getAttributes().get("oid").toString(), feature.getAttributes().get("collection_timestamp").toString())));
```
Add a new line before `new` to keep the line lengths shorter